src/goto-analyzer/show_callsites.cpp

```diff
@@ -0,0 +1,100 @@
+/*******************************************************************\
+
+Module: Show symbols that participate in shadowing
+
+Author: Norbert Manthey, nmanthey@amazon.de
+
+\*******************************************************************/
+
+#include "show_callsites.h"
+
+#include <goto-programs/compute_called_functions.h>
+#include <goto-programs/goto_model.h>
+#include <langapi/language.h>
+#include <langapi/mode.h>
+#include <util/cprover_prefix.h>
+#include <util/message.h>
+#include <util/prefix.h>
+
+#include <algorithm>
+#include <fstream>
+#include <sstream>
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+/** show_callsites writes a call graph to CSV
+ *
+ * This function creates a map between functions and consumed functions, and
+ * stores this relationship in a CSV file, to be processed outside of
+ * goto-analyzer lateron.
+ *
+ * Functions, whose address is consumed, are listed in a separate section so
+ * that outside reasoning has visibility as well.
+ **/
+
+void show_callsites(
+  const goto_modelt &goto_model,
+  message_handlert &ui,
+  const std::string &output_filename)
```

Comment:
The message handler isn't used?
We would usually call it "message_handler".

Reply:
i'll drop it, that's left over from copy&paste